website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -1,145 +1,113 @@
+<figure>
+ <img src="/img/aws_ami_create_server_status.png" style={{width: "100%", height: "auto"}} alt="A launch status screen shows the server status." />
+ <figcaption align="center"><i>A launch status screen shows the server status</i></figcaption>
+</figure>
 
-### Step 4: Deploy Appsmith On AWS Cloud
+11. Follow these steps to verify the Appsmith installation:
 
-The next step is to launch a cloud server with the Appsmith Amazon Machine Image (AMI) running on it. The AWS Console lets you do this in just a couple of clicks. Follow these steps:
+ a. On the **Launch status** screen, click the **EC2 instance id** available in the bracket.
 
-1. Navigate to the "**Amazon** **EC2 dashboard,"** and select the “**AMIs**” option in the “**Images**” menu.
-2. Search for the Appsmith Stack by entering the search term "**Appsmith**" in the search bar at the top.
-3. Select the image in the list of search results and click the “**Launch**” button.
-4. On the resulting detail page, review the available server sizes. Select the server size you wish to use and click “**Review and Launch**” to proceed.
-5. On the review page, click the “**Edit security groups**” link.
-6. On the “**Configure Security Group**” page, choose the option to “**Select an existing security group**”. Find the security group you created in Step 3 and select it. Click the “Review and Launch” button to proceed.
-7. Verify that the correct key pair (created in [**Step 2**](./aws-ami#step-2-generate-an-ssh-key-pair)) is used for the server.
-8. Confirm your selection by hitting the “**Launch Instance**” button.
+ b. Click the **Instance ID** to access the **Instance Summary** page.
 
-The AWS Console begins spinning up the new server.
+ c. Copy the **Public IPv4 address** available on the **Instance Summary** page.
 
-![Launch](/img/aws-launch.png)
+ <figure>
+ <img src="/img/aws-ecs-ami-find-DNS-to-access-appsmith.png" style={{width: "100%", height: "auto"}} alt="Use DNS or Public IP to access Appsmith." />
+ <figcaption align="center"><i>Use DNS or Public IP to access Appsmith</i></figcaption>
+ </figure>
+ 
+ d. Paste it in a browser tab and wait for the server to come up. It can take up to 5 minutes. 
+ 
+ e. Once the server is up and running, you can see the login screen of Appsmith.
 
-The process usually takes a few minutes. Use the EC2 Dashboard to check the status of the server. Once the server has launched, you can obtain its public IP address from the EC2 Dashboard, as shown below:
+ <figure>
+ <img src="/img/aws-login-page.png" style={{width: "100%", height: "auto"}} alt="Appsmith Login Screen" />
+ <figcaption align="center"><i>Appsmith Login Screen</i></figcaption>
+ </figure>
 
-![EC2 Detail](/img/aws-ec2-detail.png)
+12. Follow these steps to log into the instance using admin credentials:
```

Comment:
Can we also mention that the Appsmith instance gets initialized with default root admin user appsmith@example.com, and users have to log in as this user for the first time?

Reply:
Added default root. Wouldn't the users use as it's part of the installation guide.